SpaceGame/Assets/Scripts/MotherShipHomingDeviceUI.cs

```diff
@@ -46,7 +47,13 @@ void Update()
                 closest = dist;
             }
         }
-        
+
```

Comment:
why ? you could have simply added a single object to the list, this is purely uneccessary afaik